contracts/DAOTreasury.cdc

```diff
@@ -91,11 +95,6 @@ pub contract DAOTreasury {
       return <- vaultRef.withdraw(amount: amount)
     }
 
-    // Reference to Vault //
-    access(account) fun borrowVault(identifier: String): &FungibleToken.Vault {
-      return (&self.vaults[identifier] as &FungibleToken.Vault?)!
-    }
-
     // Public Reference to Vault //
     pub fun borrowVaultPublic(identifier: String): &{FungibleToken.Balance, FungibleToken.Receiver} {
```

Comment:
remove `FungibleToken.Receiver` so deposits must go through `depositTokens`

Reply:
Updated, please check if the change is correct